### PR TITLE
Add dates disabling/restriction props

### PR DESCRIPTION
**Pull request by NoahMLoomis**
Props added:
- minDate: The minimum date that can be selected (inlcusive)
- maxDate: The maximum date that can be selected (inlcusive)
- disabledDates: An array of date ranges to be disabled

I think having separate props for minData/maxDate and disabledDates improves clarity.

@onesine How can I update the Documentation site?

Also @JefteCaro thanks for creating the playground, it made the DX 10X better

### CODE REVIEW

**Comment by onesine** [comment]
> Props added:
> 
> * minDate: The minimum date that can be selected (inlcusive)
> * maxDate: The maximum date that can be selected (inlcusive)
> * disabledDates: An array of date ranges to be disabled
> 
> I think having separate props for minData/maxDate and disabledDates improves clarity.
> 
> @onesine How can I update the Documentation site?
> 
> Also @JefteCaro thanks for creating the playground, it made the DX 10X better

It also offers more possibilities. We can even combine everything.

**Review by onesine — COMMENTED** [review]
Can you make these changes?

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
-    "name": "react-tailwindcss-datepicker",
-    "version": "1.3.3",
+    "name": "nloomis_react-tailwindcss-datepicker",
```
[the remark this answers is not in the document]
Comment (JefteCaro) [reply package.json:2]: Thanks @NoahMLoomis 

For me, too.

All are good.
Let's just keep the latest package name and version

**Review by JefteCaro — CHANGES_REQUESTED** [review]

**Review by NoahMLoomis — COMMENTED** [review]
Finished self-review

**Review by JefteCaro — APPROVED** [review]

**Review by onesine — APPROVED** [review]

**Comment by onesine** [comment]
Merge won't happen if we don't resolve the conflicts

**Comment by JefteCaro** [comment]
@onesine Yeah

@NoahMLoomis 
Thank you for the fix.
Can I ask to have the `yarn.lock` be removed from `.gitignore` in the mean time, so the `yarn.lock` file can be added to your commits?

**Comment by onesine** [comment]
Yes, in fact, I'm working on it right now.

**Pull request merged**